app/models/concerns/income_summarisable.rb

```diff
@@ -39,6 +39,34 @@ def combined_disposable_income
     from_cfe_payload("result_summary.disposable_income.combined_total_disposable_income")
   end
 
+  def client_benefits
+    from_cfe_payload("assessment.gross_income.state_benefits.monthly_equivalents.all_sources.value")
```

Comment:
Question - are we checking the `assessment` instead of the `result_summary`, because the `result_summary` amalgamates the specific values we need?

Reply:
exactly. the assessment section returns the monthly equivalents which are what we need to diaplay, the result section only shows aggregated income figures

Reply:
Thanks for explaining - I thought as much, but was struggling to figure out how I would confim it.